code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -299,7 +299,7 @@
 
 		// Species and body temperature
 		var/datum/species/targetspecies = humantarget.dna.species
-		var/mutant = humantarget.dna.check_mutation(/datum/mutation/human/hulk) \
+		var/mutant = humantarget.dna.check_mutation(/datum/mutation/human/epilepsy) \
```

Comment:
Uhhh, why is this epilepsy now? Just remove the check_mutation call, frankly.

Reply:
> Uhhh, why is this epilepsy now? Just remove the check_mutation call, frankly.

I made it epilepsy because idk how to make it work otherwise 💀

Reply:
Remove the highlighted part.
![image](https://user-images.githubusercontent.com/106692773/234977734-67b02658-cb5d-4b57-87bc-f3ecb9fa246c.png)

Reply:
done in the last in the commit